src/main/groovy/wooga/gradle/unity/DefaultUnityPluginExtension.groovy

```diff
@@ -209,7 +216,7 @@ class DefaultUnityPluginExtension implements UnityPluginExtension {
 
     @Override
     DefaultUnityPluginExtension unityPath(Object path) {
-        customUnityPath = fileResolver.resolveLater(path)
+        this.setUnityPath(path)
```

Comment:
I still don't get why this change is in this pull request

Reply:
This is not changing any behavior, I just wanted to streamline the setter implementation for all properties in this class.

Reply:
Ah I had code duplication! Ok